packages/storage/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@mintbase-js/storage",
+  "version": "0.0.4-storage-module.0",
+  "description": "Storage functions for  Mintbase JS SDK",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc && tsc --watch & jest --watch --coverage",
+    "lint": "eslint . --fix --ext ts",
+    "test": "jest --coverage"
+  },
+  "files": [
+    "lib"
+  ],
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@bundlr-network/client": "^0.7.7",
+    "near-api-js": "^0.44.2",
+    "superagent": "^8.0.3"
```

Comment:
According to Ruben, we are using isomorphic-unfetch as our HTTP request polyfill